spring-kafka/src/main/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainer.java

```diff
@@ -156,15 +169,16 @@ protected void doStart() {
 				this.concurrency = this.partitions.length;
 			}
 			setRunning(true);
+			logger.info("Topics from ConcurrentMessageListenerContainer-->:" + (this.topics));
```

Comment:
For the future: If you agree with our comments, not reason to say "Agreed".
Just make a change silently and notify us about the push.
Otherwise we receive extra spam in our mail box. :smile: